clients/tui/nmt-page-vlan.c

```diff
@@ -35,7 +35,6 @@ G_DEFINE_TYPE (NmtPageVlan, nmt_page_vlan, NMT_TYPE_EDITOR_PAGE_DEVICE)
 #define NMT_PAGE_VLAN_GET_PRIVATE(o) (G_TYPE_INSTANCE_GET_PRIVATE ((o), NMT_TYPE_PAGE_VLAN, NmtPageVlanPrivate))
 
 typedef struct {
-	NMSettingWired *s_wired;
 
 } NmtPageVlanPrivate;
 
```

Comment:
An empty private structure will raise warnings from GLib at runtime. You should get rid of it altogether and remove the g_type_class_add_private().